assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -35,7 +35,10 @@ pageStartup.start();
 
 // ----- Redux Store ----- //
 
-const store = createStore(reducer, applyMiddleware(thunkMiddleware));
+const store = createStore(reducer, {
+  intCmp: getQueryParameter('INTCMP', 'gdnwb_copts_recurring_contributions_landing_default'),
```

Comment:
Yup, I understand now. I'm saying that I don't think we should have a default campaign code. If one hasn't been passed through in the query param then we don't set one.